applications/FluidTransportApplication/python_scripts/fluid_transport_coupling_analysis.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+## Importing modules -----------------------------------------------------------------------------------------
+
+# Import kratos core and applications
+import KratosMultiphysics as Kratos
+
+from KratosMultiphysics.FluidTransportApplication.fluid_transport_analysis import FluidTransportAnalysis
+from KratosMultiphysics.analysis_stage import AnalysisStage
+
+class FluidTransportCouplingAnalysis(FluidTransportAnalysis):
+    '''Main script for fluid + fluid transport simulations.'''
+
+    def __init__(self,model,parameters):
+
+        # Creating solver and model part and adding variables
+        super(FluidTransportCouplingAnalysis,self).__init__(model,parameters)
+
+#TODO if we update processes input in fluid transport interface, this file will not be necessary
+
+    def _CreateProcesses(self, parameter_name, initialization_order):
```

Comment:
Why do you need this method?

Reply:
(I guess it is because your `FluidTransportAnalysis` `_CreateProcesses` isn't compatible with the base stage one)